src/shared/components/oncoprint/TooltipUtils.ts

```diff
@@ -541,13 +541,30 @@ export function makeGeneticTrackTooltip(
     function listOfCNAToHTML(data: any[], multipleSamplesUnderMouse: boolean) {
         const countsMap = new ListIndexedMapOfCounts();
         for (const d of data) {
-            countsMap.increment(d.hugo_gene_symbol, d.cna, d.oncokb_oncogenic);
+            countsMap.increment(
+                d.hugo_gene_symbol,
+                d.cna,
+                d.oncokb_oncogenic,
+                d.driver_filter,
+                //FIXME Should not be part of the key
```

Comment:
whats going on here?

Reply:
I have looked at this again. All was fine afterall. The FIXME was removed.